core/primitives/block_header.hpp

```diff
@@ -25,8 +26,9 @@ namespace kagome::primitives {
      * @param extrinsics_root extrinsics root
      * @param digest digest collection
      */
-    BlockHeader(common::Hash256 parent_hash, size_t number, common::Hash256 state_root,
-                common::Hash256 extrinsics_root, Buffer digest);
+    BlockHeader(common::Hash256 parent_hash, size_t number,
```

Comment:
Please, update the docs with `@param number`

Reply:
Thanks, fixed a mistake by the way.